kits/js/ResultWidget.js

```diff
@@ -127,25 +130,14 @@ jT.ItemListWidget.prototype = {
       item.href_target = doc.s_uuid;
     } 
     else {
-      var external = "External database";
-      
-      if (doc.owner_name && doc.owner_name.lastIndexOf("caNano", 0) === 0) {
-        item.logo = "images/canano.jpg";
-        item.href_title = "caNanoLab: " + item.link;
-        item.href_target = external = "caNanoLab";
-        item.footer = '';
-      }
-      else {
-        item.logo = "images/external.png";
-        item.href_title = "External: " + item.link;
-        item.href_target = "external";
-      }
+      item.href_title = "External: " + item.link;
+      item.href_target = "external";
       
       if (doc.content.length > 0) {
```

Comment:
best to have the field content configurable, i.e. the valid field is `content_hss`

Reply:
may be this is not the case, but currently even if there is nonempty content field, the links shown are as for ambit entries

Reply:
Currently, the `redenerSubstance()` method is quite hacky, and I cannot follow the logic, behind is. The change was based on "preserve the existing functionality, removing the _caNanoLab_ special case". 
It would be great, if you can provide a structured description of what should be presented, based on the `doc` fields. Currently `tagDbs` is utilized in both the logo image, and the composition URL, which I'm not sure is exactly correct.

Reply:
1) if there is empty doc.content link show the Material, Composition, Studies links based on ambit API 
2) if there is a doc.content field (and it is a link)  show the link directly.  No material, composition or study link should be constructed in this case
3) tagDbs should be used for the logo in both 1) and 2) cases, it currently is used only in case 1
4) generating composition URL makes sense only in case 1)